### PR TITLE
Several minor edits regarding CSS colors

### PR DESCRIPTION
**Pull request by estelle**
This PR includes several little tweaks i noticed needed edits during my work on CSS colors

- CMYK function page: grammar nit and  added a see also section
- CSS colors module: added experiemental inline to colors level 6 feature. Added 2 glossary terms
- `color-contrast()` function: added a see also section
- A11y page on color and luminance: link to rgb glossary page
- `color` property: add WCAG color contrast to see also

### CODE REVIEW

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ slug: Web/Accessibility/Understanding_Colors_and_Luminance
 page-type: guide
 ---
 
+- [WCAG: color contrast](/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)
```
Comment (chrisdavidmills) [inline files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:7]: It seems a bit odd to dump this right at the top of the article. Shouldn't it go in a note below the info, or in "See also", perhaps?

**Review by chrisdavidmills — APPROVED** [review]
Looking good @estelle, just one comment for you to consider. Approving so as not to hold it up.

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:7]: that was an oops.

Comment (estelle) [inline files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:7]:
```suggestion
```

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ slug: Web/Accessibility/Understanding_Colors_and_Luminance
 page-type: guide
 ---
 
+
```
Comment (estelle) [inline files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:7]:
```suggestion
```

**Pull request merged**